lib/vm/src/store.rs

```diff
@@ -104,6 +105,19 @@ impl StoreObjects {
         }
     }
 
+    /// Return an immutable iterator over all globals
+    pub fn iter_globals(&self) -> Iter<VMGlobal> {
+        self.globals.iter()
+    }
+
+    /// Set a global, at index idx. Will panic if idx is out of range
+    pub fn set_global(&self, idx: usize, val: u128) {
+        assert!(idx >= self.globals.len());
```

Comment:
Shouldn't this be `<=` ?

In general, it seems unexpected that this API signature would panic.

Should probably return a `Result<(), ?>` instead.

Reply:
Or rather `<`.

Reply:
fixed